app/views/generic_files/_show_actions.html.erb

```diff
@@ -0,0 +1,34 @@
+<div id="show_actions">
+<h2 class="non lower">Actions</h2>
+    <p>
+      <%= render_download_link %>
+      <% if Sufia.config.analytics %>
+        &nbsp;|&nbsp;
+        <%= link_to "Analytics", sufia.stats_generic_file_path(@generic_file), id: 'stats' %>
+      <% end %>
+      <% if can? :edit, @generic_file %>
+          &nbsp;|&nbsp;
+          <%= link_to "Edit", sufia.edit_generic_file_path(@generic_file) %>
+      <% end %>
+    </p>
```

Comment:
Yeah, that's intentional in this override, to clarify. We're not using FeaturedWork, and ```public?``` forces extra Fedora traffic, slowing down the page.